lepton/image.go

```diff
@@ -223,6 +223,10 @@ func BuildPackageManifest(packagepath string, c *types.Config) (*fs.Manifest, er
 		}
 	}
 
+	for k, v := range c.ManifestPassthrough {
+		m.AddPassthrough(k, v)
+	}
```

Comment:
LGTM. Alternatively, these lines can be moved from BuildManifest() to setManifestFromConfig(), which is called by both  BuildManifest() and BuildPackageManifest().